api.md

```diff
@@ -887,12 +888,51 @@ SELECT detach_tablespaces('conditions');
 ```
 
 ---
+## distributed_exec() [](distributed_exec)
+
+This command is used on an access node to execute a SQL command across
+the data nodes of a distribute database. This command can be used to,
+e.g., create cluster-wide objects across all instances of a
+distributed database. A common use case is creating the roles and
+permissions needed in a distributed database.
+
+Note that the command is _not_ executed on the access node itself.
+
+#### Required Arguments [](distributed_exec-required-arguments)
+
+|Name|Description|
+|---|---|
+| `query` | The command to execute on data nodes. |
```

Comment:
For me it is unclear if I can chain commands here, can i runthe following for example:
```sql
SELECT distributed_exec($sql$CREATE USER davide; GRANT myapp TO davide;$sql$);
```

Reply:
This isn't currently supported. Not sure it can be supported either, but might be worth investigating. Will make clear.